catalog/opensearch/catalog-opensearch-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -51,6 +51,17 @@
             type="Boolean" default="true"
             description="Converts Polygon and Point-Radius searches to a Bounding Box for compatability with older interfaces. Generated bounding box is a very rough representation of the input geometry."/>
 
+        <AD description="Disable CN check for the server certificate. This should only be used when testing."
```

Comment:
I think that has to do with the default security settings from an out-of-the-box installation